types/chromecast-caf-receiver/index.d.ts

```diff
@@ -11,14 +11,155 @@
 /// <reference path="./cast.framework.system.d.ts" />
 /// <reference path="./cast.framework.ui.d.ts" />
 
-import { PlayerDataChangedEvent } from './cast.framework.ui';
-import { NetworkRequestInfo } from './cast.framework';
-import { Event } from './cast.framework.events';
+import framework from './cast.framework';
+import breaks from './cast.framework.breaks';
+import events from './cast.framework.events';
+import messages from './cast.framework.messages';
+import ui from './cast.framework.ui';
+
+interface CastReceiverFrameworkSDK {
```

Comment:
This was an annoying manual process but without it tsc would not detect the imported interfaces... i.e.
```
interface CastReceiverFrameworkSDK {
    framework: framework;
}
```

did not work.